doc/LICENSE.txt

```diff
@@ -3,14 +3,14 @@
 LICENSE
 =======
 
-Copyright (c) 2008--2015, Theano Development Team
+Copyright (c) 2008--2016, Theano Development Team
 All rights reserved.
 
 Contains code from NumPy, Copyright (c) 2005-2011, NumPy Developers.
 All rights reserved.
 
 Contain CnMeM under the same license with this copyright:
-Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+Copyright (c) 2016, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Not sure, but since we use CUDA, i had thought i should update NVIDIA license as well. Will revert